gvsbuild/tools.py

```diff
@@ -30,7 +30,8 @@ def __init__(self):
         Tool.__init__(
             self,
             "cargo",
-            version="stable",
+            version="1.69.0",
```

Comment:
can we add a FIXME here?

Maybe factor the version out in a constant

```
# FIXME: temporarily force 1.69.0 until <link> is fixed
# RUST_VERSION="stable"
RUST_VERSION="1.69.0"

Tool.__init__(
     ...
     version=RUST_VERSION
     ...
```

Reply:
Hey @pbor, thanks for the review!

I was thinking for this part that we will always keep a version pinned rather than using stable. I think we have learned our lesson that we need to pin this dependency like we do the others.

Reply:
I am not sure about that... I think finding out early in CI etc is actually a good thing rather than realizing 2 months late when the problem has ossified, to be honest of other projects had the ability to automatically get the latest stable, we probably would instead of updating weekly :)

No strong opinion though, if we want to threat rust updates as we do for the rest I am also ok with that

Reply:
The new updated version will show up immediately in our `gvsbuild outdated` script, and I have been bumping all the dependencies weekly. I rather get notified by the PR that something isn't compatible before we merge in.

Reply:
ok